pkg/kdump/manifest.json.in

```diff
@@ -6,7 +6,7 @@
 
     "tools": {
         "index": {
-            "label": "Kernel dump",
```

Comment:
Thanks @garrett for the commit, but I'll have to drop it and do it properly. We need to also change it in .po files and also in weblate repo, so I want script to do it and not do it manually.
I'll do it now.